### DISCUSSION TITLE
Project setup in Android Studio

### DISCUSSION THREAD

**Discussion by ltomes** [trupici/G-Watch-Wear#42] [Q&A]
Hi there,

I am looking to setup this project via Android Studio.
When I try to build `release/0.9.9` (I am using this over `master` or `dev` in case those are not in a compileable state) I get the following gradle error on sync:
```
Build file '<path>\g-watch\G-Watch-Wear\mobile\build.gradle' line: 15

A problem occurred evaluating project ':mobile'.
> path may not be null or empty string. path='null'
```

I don't see any developer setup instructions, but I'm happy to take anything I learn here to an MR for future developers.

Thanks for your time!

**Comment by Cleboost** [comment]
I have same problem

**Comment by marconucara** [comment]
I solved removing from both build.gradle files the signingConfigs, signingConfig and localazy sections. Just remember to not push them.
